libraries/standard/mqtt/src/mqtt.c

```diff
@@ -1011,6 +1011,7 @@ MQTTStatus_t MQTT_Connect( MQTTContext_t * pContext,
     {
         LogInfo( ( "MQTT connection established with the broker." ) );
         pContext->connectStatus = MQTTConnected;
+        pContext->keepAliveIntervalSec = pConnectInfo->keepAliveSeconds;
```

Comment:
Was this a bug?!

Reply:
Yes

Reply:
Is there a missing expectation for it in the unit tests?

Reply:
What do you mean?